site/docs/design_tokens/box-shadow.mdx

```diff
@@ -0,0 +1,39 @@
+---
+name: Box-shadow
+menu: Design tokens
+route: /design-tokens/box-shadow
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Box-shadow
+
+<LargeParagraph>Box-shadows are used to cast shadows from interface elements to backgrounds or other elements in order to allow visual cohesion and a more accessible experience.</LargeParagraph>
+
+Shadows under elements can be used to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or otherwise remain unnoticed.
+
+## Principles
+- **Shadows are not used to signify elevation in HDS.** Elements that require distinction can have shadows applied.
```

Comment:
Should the elevation be one thing that requires distinction?
Something like Shadows are only used to make a difference in elements. Elements having a clear hierarchy over other elements can use shadows, bringing element more prominent.

Reply:
Koitin tässä juuri selittää ettei käytetä ns. korkeuserojen jäljittelyyn, vaan vain silloin kun kaksi elementtiä vaativat erottelua toisistaan.

Reply:
Also, adding the elevation as "a thing" in HDS requires a very detailed description of the z-index etc. As we only use the shadow (atleast for now) to differentiate and not elevate, I fail to see the point.